zhmcclient/testutils/_cpc_fixtures.py

```diff
@@ -57,10 +57,12 @@ def all_cpcs(request, hmc_session):  # noqa: F811
     The CPCs to test against are defined in the ``cpcs`` variable for the
     HMC entry in the :ref:`HMC inventory file`.
 
-    A test function parameter using this fixture resolves to a list of
-    :class:`zhmcclient.Cpc` objects representing that set of CPCs. These
+    A test function parameter with the name of this fixture resolves to a list
+    of :class:`zhmcclient.Cpc` objects representing that set of CPCs. These
     objects have the "short" set of properties from :meth:`zhmcclient.Cpc.list`.
 
+    The test function is repeated for each CPC in that set of CPCs.
```

Comment:
Well, this doesn't happen automatically, right? 
The developer has to write the corresponding code, so this statement here is more like guidance how tests should be written?! If so, that could be made more clear.

Reply:
That is a good catch - copy & paste error. In this case, the fixture returns a list, and the test function needs to iterate through the list (or pick one). Will fix it.

Reply:
DONE